avm/src/vm/telemetry.rs

```diff
@@ -27,15 +27,11 @@ pub async fn log(event: &str) {
   });
   let client =
     Client::builder().build::<_, Body>(hyper_rustls::HttpsConnector::with_native_roots());
-  if client
+  let _ = client
```

Comment:
What's the assignment here for?

Reply:
it ignores the `Result` returned, which is what we were doing before, but I can change it back?

Reply:
I was actually better with your refactoring, but I didn't understand why we needed the assignment at all. Just removing the `let _ =` means it won't compile?